Dockerfile

```diff
@@ -1,13 +1,16 @@
-FROM ruby:3.2.2
+FROM ruby:3.2.2-alpine
 
 LABEL org.opencontainers.image.source=https://github.com/searls/feed2gram
 LABEL org.opencontainers.image.description="Reads an Atom feed and posts its entries to Instagram (basically feed2toot, but for Instagram)"
 LABEL org.opencontainers.image.licenses=GPLv3
 
 WORKDIR /srv
-COPY Gemfile Gemfile.lock feed2gram.gemspec .
+COPY Gemfile Gemfile.lock feed2gram.gemspec ./
```

Comment:
Curious, what does this `/` do?

Reply:
It's for compatibility with non-buildkit `docker build`. In the "before times" if you copied multiple files, the final argument _must_ end with a `/` or it will fail.

It might not come up in practice, but I'm sure that somebody out there is using a really old docker and will appreciate not having to fix this incompatibility.